registration/forms.py

```diff
@@ -61,6 +61,49 @@ def validate(self, value):
         except django.core.exceptions.ValidationError:
             raise django.core.exceptions.ValidationError(self.message, code=self.code)
 
+# This class was built to validate License ID format in server-side
```

Comment:
@mabuaisha Do you think that we will get back to this method in the future? If no and you don't need it please remove it.